wfe/web-front-end.go

```diff
@@ -289,15 +288,16 @@ func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool) ([]
 		return nil, nil, reg, errors.New("No body on POST")
 	}
 
-	body, err := ioutil.ReadAll(request.Body)
+	bodyBytes, err := ioutil.ReadAll(request.Body)
 	if err != nil {
 		return nil, nil, reg, err
 	}
 
+	body := string(bodyBytes)
 	// Parse as JWS
-	parsedJws, err := jose.ParseSigned(string(body))
+	parsedJws, err := jose.ParseSigned(body)
 	if err != nil {
-		wfe.log.Debug(fmt.Sprintf("Parse error reading JWS: %v", err))
+		wfe.log.Debug(fmt.Sprintf("Parse error reading JWS: %#v", err))
```

Comment:
So, I've had trouble in the past with errors having more useful context in their structure than in the String().

But I'm not beholden to that! It's one of my quirks.